usr/share/rear/rescue/default/500_ssh.sh

```diff
@@ -1,50 +1,53 @@
-# 500_ssh.sh
-#
-# take ssh for Relax-and-Recover
+# SSH client (ssh) and server (sshd) configuration for Relax-and-Recover
 #
 # This file is part of Relax-and-Recover, licensed under the GNU General
 # Public License. Refer to the included COPYING for full text of license.
 
-if has_binary sshd; then
+if has_binary ssh || has_binary sshd; then
```

Comment:
You can do `has_binary ssh || has_binary sshd || return` to bail out early to skip SSH treatment. That will save you another `if` and make the script easier to read.

Reply:
True. This structure was just inherited. It's OK for me to change this.